app/src/main/java/com/android/sample/ui/library/Library.kt

```diff
@@ -92,28 +82,25 @@ fun LibraryScreen(navigationActions: NavigationActions) {
                         Modifier.size(28.dp).graphicsLayer(alpha = 0.99f).drawWithCache {
                           onDrawWithContent {
                             drawContent()
-                            drawRect(PrimaryGradientBrush, blendMode = BlendMode.SrcAtop)
+                            drawRect(IconsGradientBrush, blendMode = BlendMode.SrcAtop)
                           }
                         })
               }
             },
             modifier =
                 Modifier.shadow(
-                        elevation = 2.dp,
-                        spotColor = Color(0x1A000000),
-                        ambientColor = Color(0x1A000000))
-                    .border(width = 1.dp, color = Color(0xFFEADDFF))
+                        elevation = 2.dp, spotColor = ShadowColor, ambientColor = ShadowColor)
+                    .border(width = 1.dp, color = BorderColor)
                     .width(412.dp)
                     .height(48.dp)
-                    .background(color = Color(0xFFFFFFFF)),
+                    .background(color = PrimaryWhite),
```

Comment:
Consider using the colors from the MaterialTheme.colorScheme to maintain consistency.

```suggestion
                    .background(color = MaterialTheme.colorScheme.background),
```